max_obstacle_height: 0.60
obstacle_range: 2.5
raytrace_range: 3.0
footprint_padding: 0.0
robot_radius: 0.15
global_frame: odom
robot_base_frame: base_footprint
transform_tolerance: 2.0

local_costmap:
  update_frequency: 2.0
  publish_frequency: 1.0
  static_map: false
  rolling_window: true
  width: 3.5
  height: 3.5
  obstacle_max_range: 6.0
  raytrace_max_range: 6.0
  resolution: 0.02
  global_frame: odom
  plugins:
    #- { name: obstacle_layer, type: 'costmap_2d::VoxelLayer' }
    - {name: scan_obstacle_layer, type: 'costmap_2d::ObstacleLayer'}
    - {name: bump_obstacle_layer, type: 'costmap_2d::ObstacleLayer'}
    #     - {name: footprint_layer, type: 'costmap_2d::FootprintLayer'}
    - { name: inflation_layer, type: 'costmap_2d::InflationLayer' }


  inflation_layer: {cost_scaling_factor: 10.0, enabled: true, inflation_radius: 0.45, robot_radius: 0.34}

  scan_obstacle_layer:
    observation_sources: scan
    scan: {clearing: true, data_type: LaserScan, marking: true, inf_is_valid: false, obstacle_range: 4.0, topic: /scan}
    track_unknown_space: false

  bump_obstacle_layer:
    bump: {clearing: false, data_type: PointCloud2, marking: true, topic: /mobile_base/sensors/bumper_cloud, observation_persistence: 0.0,
           observation_timeout_from_now: 30.0, expected_update_rate: 1.0}
    observation_sources: bump
    track_unknown_space: false

